Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutorInterface.h

```diff
@@ -22,13 +22,16 @@ namespace AzToolsFramework
             AZ_RTTI(InstanceUpdateExecutorInterface, "{26BB4ACF-6EA5-4668-84E7-AB75B90BA449}");
             virtual ~InstanceUpdateExecutorInterface() = default;
 
-            // Add all Instances of Template with given Id into a queue for updating them later.
+            // Add an instance into queue for updating it later.
+            virtual void AddInstanceToQueue(InstanceOptionalReference instance) = 0;
+
+            // Add all instances of template with given id into queue for updating them later.
```

Comment:
nit: While you are fixing this, might as well change it to use Doxygen markup (//!) and use @param and @return tags.

Reply:
Thanks for pointing out this. I will update comments for these classes. I will update comments in other classes in next PR.